src/CommitmentsV2/SFA.DAS.CommitmentsV2.Api/Controllers/DraftApprenticeshipController.cs

```diff
@@ -92,5 +103,26 @@ public async Task<IActionResult> Add(long cohortId, [FromBody]AddDraftApprentice
                 DraftApprenticeshipId = result.Id
             });
         }
+
+        [HttpPost]
+        [Route("{apprenticeshipId}")]
+        public async Task<IActionResult> Delete(long cohortId, long apprenticeshipId, [FromBody]DeleteDraftApprenticeshipRequest request)
+        {
+            try
+            {
+                var command = await _deleteDraftApprenticeshipsMapper.Map(request);
+                command.CohortId = cohortId;
+                command.ApprenticeshipId = apprenticeshipId;
+
+                await _mediator.Send(command);
+
+                return Ok();
+            }
+            catch (Exception exception)
+            {
+                var excep = exception.Message;
+                return NotFound();
```

Comment:
Not sure this is the correct way of dealing with the exception. We have a framework in place that will pass model/domain errors back to the client. 

This catch will consume the error within the API and return a 404 and the client code will then have to 'guess' what happened.

Reply:
Oops, this was something I was messing around locally, erroneously checked in. But totally Agree, This is not a correct way of handling exception.